docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -443,6 +614,12 @@ In most cases, the same extended comparators are available for both lifting and
 - `Filter by Proximity (*near=)`_
 - `Filter by (Arrays) (*[ ])`_
 
+Each extended comparision operator can be used with any kind of property (primary, secondary, universal, or
+extended) whose :ref:`gloss-type` is appropriate for the comparison used. When filtering by secondary property
+value, you can optionally specify an :ref:`interface` name to filter based on all forms that inherit that
```

Comment:
This seems like it should be referencing an interface property, but sounds like it is just talking about doing a filter on things that have an interface?

Reply:
Revised wording to (try to?) clarify